xc/device.ml

```diff
@@ -1986,6 +1986,11 @@ module Backend = struct
               )
 
           let qmp_event_thread () =
+            (* Add the existing qmp sockets first *)
+            List.iter (
+              fun x -> try Scanf.sscanf x "qmp-event-%d" (fun x -> add x) with _ -> ()
```

Comment:
`fun x -> add x` can be written as `add`.

Reply:
Updated, thanks.